### PR TITLE
Added items text product det

### PR DESCRIPTION
**Pull request by Farhuts**
added text indicating that items were added to the card

### CODE REVIEW

File: client/components/user-home.js [hunk 1]
```diff
@@ -7,10 +7,18 @@ import {connect} from 'react-redux'
  */
 export const UserHome = props => {
   const {email} = props
-
+  console.log(props)
```
Comment (scoy028) [inline client/components/user-home.js]: delete console log
```suggestion
```

File: client/components/product-details.js [hunk 2]
```diff
@@ -33,10 +41,14 @@ class ProductDetails extends Component {
   }
 
   render() {
+    console.log('VALUE ', this.state.value)
```
Comment (scoy028) [inline client/components/product-details.js]: delete console log
```suggestion
```

File: client/components/order.js [hunk 3]
```diff
@@ -5,25 +5,64 @@ import {getUserOrderThunk} from '../store/order'
...
+              Quantity: <br />
+              <div className="input-field col s12 left">
+                <select className="browser-default order">{list}</select>
+              </div>
+              <br />
+            </p>
+          </div>
+        )
       })
     console.log('ORDER CART, ', this.props.currentOrder)
```
Comment (scoy028) [inline client/components/order.js]: delete console log
```suggestion
```

**Review by scoy028 — CHANGES_REQUESTED** [review]
just need to delete some console logs; left comments marking them

**Review by Farhuts — COMMENTED** [review]
Thank you

**Review by scoy028 — DISMISSED** [review]
👍

**Review by halimchu — APPROVED** [review]

**Pull request merged**